src/mapping/query/functions.rs

```diff
@@ -280,20 +280,53 @@ impl Function for UpcaseFn {
 
 #[derive(Debug)]
 pub(in crate::mapping) struct DowncaseFn {
-    query: Box<dyn Function>,
+    path: Box<dyn Function>,
+    append: Option<Box<dyn Function>>,
```

Comment:
This is a nonsensical change to support an example test case added to the `remap.toml` behaviour test. Please ignore the functionality of this function for now.